### ISSUE TITLE
Help please! doesnt work!

### ISSUE THREAD

**Issue by riskfaktor** [Guichaguri/DiscordDJ#45]
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'start' ]
2 info using npm@3.6.0
3 info using node@v5.7.0
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle discord-dj@0.0.4~prestart: discord-dj@0.0.4
6 silly lifecycle discord-dj@0.0.4~prestart: no script for prestart, continuing
7 info lifecycle discord-dj@0.0.4~start: discord-dj@0.0.4
8 verbose lifecycle discord-dj@0.0.4~start: unsafe-perm in lifecycle true
9 verbose lifecycle discord-dj@0.0.4~start: PATH: C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin;C:\Users\exqui\Downloads\DiscordDJ-master\node_modules\.bin;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\ProgramData\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files (x86)\Skype\Phone\;C:\Program Files\nodejs\;C:\Users\exqui\AppData\Roaming\npm
10 verbose lifecycle discord-dj@0.0.4~start: CWD: C:\Users\exqui\Downloads\DiscordDJ-master
11 silly lifecycle discord-dj@0.0.4~start: Args: [ '/d /s /c', 'node ./runtime/DiscordDJ.js' ]
12 silly lifecycle discord-dj@0.0.4~start: Returned: code: 1  signal: null
13 info lifecycle discord-dj@0.0.4~start: Failed to exec start script
14 verbose stack Error: discord-dj@0.0.4 start: `node ./runtime/DiscordDJ.js`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\lifecycle.js:232:16)
14 verbose stack     at emitTwo (events.js:100:13)
14 verbose stack     at EventEmitter.emit (events.js:185:7)
14 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\spawn.js:24:14)
14 verbose stack     at emitTwo (events.js:100:13)
14 verbose stack     at ChildProcess.emit (events.js:185:7)
14 verbose stack     at maybeClose (internal/child_process.js:827:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
15 verbose pkgid discord-dj@0.0.4
16 verbose cwd C:\Users\exqui\Downloads\DiscordDJ-master
17 error Windows_NT 10.0.10586
18 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
19 error node v5.7.0
20 error npm  v3.6.0
21 error code ELIFECYCLE
22 error discord-dj@0.0.4 start: `node ./runtime/DiscordDJ.js`
22 error Exit status 1
23 error Failed at the discord-dj@0.0.4 start script 'node ./runtime/DiscordDJ.js'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the discord-dj package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     node ./runtime/DiscordDJ.js
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs discord-dj
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls discord-dj
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]

**Comment by Guichaguri** [comment]
First of all, download the bot [here](https://github.com/Guichaguri/DiscordDJ/releases)
You have to install the bot, it seems like it's not installed correctly.

**Comment by riskfaktor** [comment]
yeah tried your link same thing as before, it installs, then asks if I want to start the bot i press y+enter and then the command promt closes

**Comment by riskfaktor** [comment]
and when I try starting it with start-bot it gives me some error code.

fs.js:584
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open 'C:\Users\exqui\Downloads\windows-with-ffmpeg\node_modules\discordie\lib\core..\deps\cas\addtrustexternalcaroot.crt'
    at Error (native)
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.readFileSync (fs.js:431:33)
    at Array.module.exports.addFile (C:\Users\exqui\Downloads\windows-with-ffmpeg\node_modules\ssl-root-cas\ssl-root-cas.js:4518:30)
    at C:\Users\exqui\Downloads\windows-with-ffmpeg\node_modules\discordie\lib\core\SSLCA.js:10:7
    at Array.forEach (native)
    at Object.<anonymous> (C:\Users\exqui\Downloads\windows-with-ffmpeg\node_modules\discordie\lib\core\SSLCA.js:9:9)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
 Press any key to continue . . .

**Comment by Guichaguri** [comment]
It's something to do with Node.js. Try to update it

**Comment by Shirafune9** [comment]
I have the same problem as the person above. My node.js is up to date, I've downloaded it from the official site, it even says the version is 5.7.

**Comment by ciruz** [comment]
I'm experiencing the same problem with a new installed Bot. After setting up the bot, he crashed with the exact same error.

But i found out whats the problem @Guichaguri.

``lib/Audio/SoundcloudAudio.js`` is missing, after copying it from master (https://raw.githubusercontent.com/Guichaguri/DiscordDJ/master/lib/Audio/SoundcloudAudio.js) into the correct directory, I started the bot and everything worked fine.

When i copy this file, before I start the bot setup, the setup works fine and nothing crashes.

So basically the only problem is, that this file is missing in the ``0.0.4`` ``windows-with-ffmpeg`` release, if you use without ffmpeg release the file is there.

**Comment by Guichaguri** [comment]
Thanks. I'll release a new version tomorrow.

**Comment by Guichaguri** [comment]
Fixed in [v0.0.5](https://github.com/Guichaguri/DiscordDJ/releases/tag/v0.0.5)
